DEVELOPING.md

```diff
@@ -64,6 +64,28 @@ From either your VM or Mac:
 * `hab studio enter`
 * `start-builder`
 
+### Merging the shards
+
+If you created your development environment before August 17th 2018, you have
+a database that contains 128 shards. You need to merge them into one, or your
+services will flap and spew errors into the log that say "Shard migration
+hasn't been completed successfully".
+
+To migrate your shards, run the following commands from inside your development
+studio:
+
+
+```
+PGPASSWORD=$(cat /hab/svc/builder-datastore/config/pwfile) tools/merge-shards/merge-shards.sh jobsrv migrate
```

Comment:
Do we need to include a manual step for the sequence id for jobsrv?

Reply:
Done